code/frameworks/preact-vite/package.json

```diff
@@ -0,0 +1,83 @@
+{
+  "name": "@storybook/preact-vite",
+  "version": "7.0.0-beta.15",
+  "description": "Storybook for Preact and Vite: Develop Preact components in isolation with Hot Reloading.",
+  "keywords": [
+    "storybook"
+  ],
+  "homepage": "https://github.com/storybookjs/storybook/tree/main/frameworks/preact-vite",
+  "bugs": {
+    "url": "https://github.com/storybookjs/storybook/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/storybookjs/storybook.git",
+    "directory": "frameworks/preact-vite"
+  },
+  "funding": {
+    "type": "opencollective",
+    "url": "https://opencollective.com/storybook"
+  },
+  "license": "MIT",
+  "exports": {
+    ".": {
+      "require": "./dist/index.js",
+      "import": "./dist/index.mjs",
+      "types": "./dist/index.d.ts"
+    },
+    "./preset": {
+      "require": "./dist/preset.js",
+      "import": "./dist/preset.mjs",
+      "types": "./dist/preset.d.ts"
+    },
+    "./package.json": {
+      "require": "./package.json",
+      "import": "./package.json",
+      "types": "./package.json"
+    }
+  },
+  "main": "dist/index.js",
+  "module": "dist/index.mjs",
+  "types": "dist/index.d.ts",
+  "files": [
+    "dist/**/*",
+    "types/**/*",
+    "README.md",
+    "*.js",
+    "*.d.ts"
+  ],
+  "scripts": {
+    "check": "../../../scripts/node_modules/.bin/tsc --noEmit",
+    "prep": "../../../scripts/prepare/bundle.ts"
+  },
+  "dependencies": {
+    "@preact/preset-vite": "^2.5.0",
+    "@rollup/pluginutils": "^4.2.0",
```

Comment:
This dependency doesn't seem to be used, and can be removed.  In react-vite, it's used for the docgen plugin.  How is docgen handled here for preact?

The same goes for `ast-types` and `magic-string`, they can be removed if we don't need a docgen plugin.

Reply:
Will remove, and add the deps once the docgen is implemented.

Reply:
I see you commented there as well, but for anyone else following along, preact docgen is currently missing.  Here's an issue to track it: https://github.com/storybookjs/storybook/issues/19739